azurerm/resource_arm_virtual_machine_scale_set.go

```diff
@@ -37,6 +37,29 @@ func resourceArmVirtualMachineScaleSet() *schema.Resource {
 
 			"zones": zonesSchema(),
 
+			"identity": {
```

Comment:
can we add an acceptance test covering this use-case?

Reply:
Done